ui/src/common/components/RouteOrRedirect.tsx

```diff
@@ -17,7 +17,7 @@ function RouteOrRedirect({
     <Route
       {...rest}
       render={props =>
-        condition ? <Component {...props} /> : <Redirect to={redirectTo} />
+        condition ? <Component {...props} /> : <Redirect to={redirectTo || '/'} />
```

Comment:
🤔 why do we need that?

Reply:
So recently I fixed almost all console errors from external libraries, but missed this one (property to is required in <Redirect /> component from connected-react-router but we sometimes pass undefined which creates error in console). I encountered it while developing so I added this simple fix additionally.